package.json

```diff
@@ -60,5 +61,11 @@
   "bugs": {
     "url": "https://github.com/niksy/stylelint-no-restricted-syntax/issues"
   },
-  "homepage": "https://github.com/niksy/stylelint-no-restricted-syntax#readme"
+  "homepage": "https://github.com/niksy/stylelint-no-restricted-syntax#readme",
+  "jest": {
```

Comment:
@vyobukhov why do we need Jest? I don’t recall that Stylelint forces testing library. See how I did it in one of my [recent plugins](https://github.com/niksy/stylelint-no-unresolved-module/blob/master/test/util/index.js)